radixdlt-java/src/main/java/com/radixdlt/client/dapps/messaging/RadixMessaging.java

```diff
@@ -1,50 +0,0 @@
-package com.radixdlt.client.dapps.messaging;
-
-import com.radixdlt.client.application.translate.data.DecryptedMessage;
-import java.util.Objects;
-
-import org.radix.utils.RadixConstants;
-
-import com.radixdlt.client.application.RadixApplicationAPI;
-import com.radixdlt.client.application.RadixApplicationAPI.Result;
-import com.radixdlt.client.atommodel.accounts.RadixAddress;
-
-import io.reactivex.Observable;
-import io.reactivex.observables.GroupedObservable;
-
-/**
- * High Level API for Instant Messaging. Currently being used by the Radix Android Mobile Wallet.
- */
-public class RadixMessaging {
```

Comment:
Why was this removed? Are there plans to reintroduce this in some other form?

Reply:
all functionality here is available via `RadixApplicationAPI`